collector/analyzer/network/protocol/dubbo/dubbo_parser.go

```diff
@@ -0,0 +1,48 @@
+package dubbo
+
+import "github.com/Kindling-project/kindling/collector/analyzer/network/protocol"
+
+const (
+	// Zero : byte zero
+	Zero = byte(0x00)
+
+	// magic header
+	MAGIC_HIGH = byte(0xda)
+	MAGIC_LOW  = byte(0xbb)
+
+	// message flag.
+	FLAG_REQUEST = byte(0x80)
+	FLAG_TWOWAY  = byte(0x40)
+	FLAG_EVENT   = byte(0x20) // for heartbeat
+	SERIAL_MASK  = 0x1f
+
+	ASCII_LOW     = byte(0x20)
+	ASCII_HIGH    = byte(0x7e)
+	ASCII_REPLACE = byte(0x2e) // .
```

Comment:
Since all constants of other parsers are in camel-case format, we better change these names to make things consistent.

Reply:
done